### DISCUSSION TITLE
2010/12/22/%E5%83%8F%E7%88%B8%E7%88%B8%E8%BF%98%E6%98%AF%E5%83%8F%E5%A6%88%E5%A6%88

### DISCUSSION THREAD

**Discussion by giscus** [ruanqizhen/blog#309] [Announcements]
# 2010/12/22/%E5%83%8F%E7%88%B8%E7%88%B8%E8%BF%98%E6%98%AF%E5%83%8F%E5%A6%88%E5%A6%88

不论哪家生了小宝，总免不了比较一下小宝到底像爸爸多还是像妈妈多。我们也一样，虽然豆豆还太小，有些特征看不出，但有些特征已经可以明显区分出来了。

https://blog.qizhen.xyz/2010/12/22/%E5%83%8F%E7%88%B8%E7%88%B8%E8%BF%98%E6%98%AF%E5%83%8F%E5%A6%88%E5%A6%88

**Comment by ruanqizhen** [comment]
之前博客下的留言：

ian f k
2010年12月23日 14:21 编辑
Lovely… Congratulations! It is good feeling to be dad :)

回复
